applications/ecallmgr/src/ecallmgr_fs_xml.erl

```diff
@@ -590,10 +595,20 @@ get_channel_params_fold(Key, Val) ->
             {Prefix, ecallmgr_util:maybe_sanitize_fs_value(Key, Val)}
     end.
 
--spec get_custom_sip_headers(wh_json:object()) -> wh_json:json_proplist().
+-spec get_custom_sip_headers(wh_json:object() | wh_proplist()) -> wh_json:json_proplist().
+get_custom_sip_headers([_|_]=Props) ->
+    [normalize_custom_sip_header_name(P) || P <- props:filter(fun is_custom_sip_header/1, Props)];
 get_custom_sip_headers(JObj) ->
     wh_json:to_proplist(wh_json:get_value(<<"Custom-SIP-Headers">>, JObj, wh_json:new())).
 
+-spec normalize_custom_sip_header_name(any()) -> any().
+normalize_custom_sip_header_name({<<"variable_", K/binary>>, V}) -> {K, V};
+normalize_custom_sip_header_name(A) -> A.
+
+-spec is_custom_sip_header(any()) -> boolean().
+is_custom_sip_header({<<"variable_sip_h_X-", _/binary>>, _}) -> 'true';
+is_custom_sip_header(_) -> 'false'.
+
```

Comment:
yes indeed it's a duplicate of those functions. Because I need headers to be prefixed with `sip_h_` but `ecallmgr_util` function is using `normalize_custom_sip_header_name` to normalize them. With the discussion that I had with @k-anderson, he seems to happy with the clone version.